### PR TITLE
Feature/reply tweet

### PR DESCRIPTION
**Pull request by Barylo**
When you click on reply button (chat icon)
image
you get the next modal to make your reply
image

### CODE REVIEW

File: src/containers/ModalTweetReplyForm.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Chat } from "react-bootstrap-icons";
+
+import TweetReplyForm from "../components/TweetReplyForm";
+import ModalUniversal from "./ModalUniversal";
+
+const ModalTweetReplyForm: React.FC = () => {
+  const TweetRepForm = <TweetReplyForm />;
```
Comment (ArtemLigerko) [inline src/containers/ModalTweetReplyForm.tsx]: propose delete TweetRepForm constant and add <TweetReplyForm /> to <ModalUniversal .../> as '{ '<TweetReplyForm /'> }'

**Comment by ArtemLigerko** [comment]
Nice working, however propose some code optimizations:
1.  Move modal logic from ModalTweetReplyForm.tsx in TweetReplyForm.tsx. example of implementation you can see in LoginForm.tsx container.
2. Add componenet TweetReplyForm.tsx in SingleTweet and FeedSingleTweet conponents straitly bypassing ReplyTweetButton.

This changes allows delete ModalTweetReplyForm.tsx and ReplyTweetButton/index.tsx

If you will, I can push commit with changes

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (alexandrtovmach) [inline .vscode/settings.json]: unnecessary file, please remove

**Review by alexandrtovmach — APPROVED** [review]
Generally, I can understand your point about continuous improvement, but I can't agree with it for 100%. Merging this PR, means that someone else should fix it instead of you, which is not fair I suppose. Specifically in this case, this part will be redone, to connect with backend, so we're fine as it is, but it's not mean, that's normal practice to create PR with uncompleted things. Just check how many improvements @NikPaukov does after Виталий  reviews 😄 

We're learning here, and not only coding, but also team playing.

Fix my small comment, and feel free to merge, but I'd see it completed in a right way, to be fair with the team. Especially, it's not drastic change, and shouldn't take more than 2 hours

File: src/components/FeedSingleTweet.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { Popover, OverlayTrigger, Button } from "react-bootstrap";
 import {
-  Chat,
+  // Chat,
   // PatchCheckFill,
```
Comment (KirillSerg) [inline src/components/FeedSingleTweet.tsx:3]: i think you can remove it permanently

**Review by KirillSerg — APPROVED** [review]
looks good

**Pull request merged**